frontend/docker/dev/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN . ~/.nvm/nvm.sh && npm install -g npm yarn
 
 COPY package.json yarn.lock ./
 
-RUN . ~/.nvm/nvm.sh && yarn install
+RUN . ~/.nvm/nvm.sh && yarn install --network-timeout 1000000
 
```

Comment:
I think this is not a common situation. Maybe we could add the timeout parameter when we need instead of adding it here for all the users.

Reply:
yeah I can remove it, for me I couldn't build it without this param

Reply:
@LEUNGUU removed that change, thx!